include/msvc/sys/socket.h

```diff
@@ -51,10 +51,10 @@ inline int readv(t_socket sock, struct iovec *iov, int nvecs)
   return -1;
 }
 
-inline int close(t_socket sock)
-{
-  return closesocket(sock);
-}
+//inline int close(t_socket sock)
```

Comment:
Are you sure that this works on all versions of VS2019? 
Because it does build on earlier versions of VS without this patch.

A much safer change would be to have #ifdefs that just disables this for VS2019 or later.